samples/matter/common/src/bridge/ble_connectivity_manager.cpp

```diff
@@ -212,6 +215,23 @@ void BLEConnectivityManager::DisconnectionHandler(bt_conn *conn, uint8_t reason)
 	}
 }
 
+#ifdef CONFIG_BRIDGE_FORCE_BT_CONNECTION_PARAMS
+/* This callback accepts or declines conn params requested by the peripheral */
+bool BLEConnectivityManager::ParamChangeRequestHandler(struct bt_conn *conn, struct bt_le_conn_param *param)
```

Comment:
Do we really need two methods looking the same for that? Can't we create some helper metod like `CheckParamChangeRequest` implementing the condition logic and then call it in `Connect` and `ParamChangeRequestHandler`?